deploy/web_scripts/notify_build.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# Accept a web-hook from GitHub telling us about
+# a new built version of Hydrant.
+
+import json, requests
+
+from sys import stdin, stdout
+from os import environ, path
+from hmac import digest
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+
+LOCKER_DIR = "/afs/sipb.mit.edu/project/hydrant/"
+
+OUTPUT_DIR = LOCKER_DIR + "web_scripts/hydrant/"
+
+CI_SECRETS_DIR = LOCKER_DIR + "ci_secrets/"
+HASH_SECRET = CI_SECRETS_DIR + "hash_secret"
+GITHUB_TOKEN = CI_SECRETS_DIR + "github_token"
```

Comment:
Arrant pedantry here, but perhaps we should use path.join()?

Reply:
you're right, we should!